docs/containers-policy.json.5.md

```diff
@@ -32,6 +32,7 @@ Policy requirements can be defined for:
   such a most specific scope define namespaces of matching images.
 - A default policy for a single transport, expressed using an empty string as a scope
 - A global default policy.
+- A wildcard entry for all subdomains.
```

Comment:
This should go into the “## Supported transports and their scopes” section, it’s not an universal mechanism (the list above basically matches `requirementsForImageRef`)